pyam/units.py

```diff
@@ -26,21 +26,24 @@ def convert_unit(df, current, to, factor=None, registry=None, context=None,
     # Convert using a pint.UnitRegistry; default the one from iam_units
     registry = registry or iam_units.registry
 
+    # Make versions without -equiv
```

Comment:
I think that having an own function for this simple removal is a bit excessive, you could do that in one line with 
```
current, _to = [i.replace('-equiv', 'e') for i in [current, to]]
```
(note the `_to` as an alternative to kee p `to` for the renaming)

Reply:
By leaving out the 'e' this works for all units, even ones without the alias set up.